src/App.js

```diff
@@ -14,11 +15,137 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    isVisible: true,
```

Comment:
```suggestion
    isVisible: false,
```
Изначально список не показывается